components/CommentCard.tsx

```diff
@@ -0,0 +1,52 @@
+import style from "@/components/CommentCard.module.css";
+import Best from "@/image/icons/ic_best.svg";
+import Heart from "@/image/icons/ic_heart.svg";
+
+interface Comment {
+  updateAt: string;
+  createdAt: string;
+  likeCount: number;
+  image: string;
+  content: string;
+  title: string;
+  nickname: string;
+  id: number;
+}
+
+interface CommentProps {
+  comments: Comment[];
+  showBest?: boolean;
+}
+
+const CommentCard: React.FC<CommentProps> = ({
```

Comment:
`map`을 이용해서 렌더링하고 있으니까 복수의 코멘트카드임을 나타내주면 좋겠습니다.

```suggestion
const CommentCards: React.FC<CommentProps> = ({
```